proposals/drafts/ocdid_governance.rst

```diff
@@ -0,0 +1,140 @@
+
+``Contributor``
+~~~~~~~~~~~~~~~
+Any individual or organization ``user`` may become a ``contributor`` by submitting a pull request adding, correcting, or aliasing jurisdictional OCDIDs.
+First time ``contributors`` will be advised via a CONTRIBUTING.RST to consult with an existing ``committer`` from their country before submitting their first PR.
+
+``Committer`` 
+~~~~~~~~~~~~~
+``Committers`` have the ability to approve and commit pull requests from ``contributors`` and other ``committers`` within their geographic scope.
+
+* Unless otherwise specified, ``committers`` will be approved for and by their country of residence/expertise. All responsibilities and privileges associated with being a ``committer`` are limited to issues within that country's community.
+* A small subset of global ``committers`` will be established, for the sake of international coordination, whose participation may be requested in all votes, regardless of country. These global ``committers`` will not be subject to the responsibilities of communities outside their own country unless otherwise specified.
+* Government officials from sub-country jurisdictions who wish to become ``committers`` may choose to limit their scope to their particular jurisdiction, and thus would not be expected to participate in wider conversations or actions.
+* Having ``committer`` authority in a country does not entitle an individual to any authority outside that country. Any ``committer`` wishing to contribute outside of their geographic scope MUST adhere to the processes and procedures of that country's community. Failure to do so may result in the revocation of ``committer`` status.
+* Country-based communities may, through consent of their community's ``committers``, choose to adapt this governance model and its processes to better suit their needs.
+
+An initial cohort of ``committers`` will be determined by current project contributors. A public list of ``committers`` will be maintained in a separate OCDEP.
+
+A separate ``committers`` communication channel for each country-based community will be established. A list of each country's ``committer`` email addresses will be maintained and email will be used to conduct official actions unless otherwise specified. 
+
+Any individual who is a ``contributor`` and who agrees to the responsibilities listed here may request to become a ``committer``via a country's agreed upon communication channel. That request will then be shared with the group of that country's current ``committers`` for approval. 
+
+* There will be a two month period where existing ``committers`` may raise objections to a request, otherwise the request will be considered approved.
+* In the case that an objection is raised, a discussion and vote will be had amongst the country's existing ``committers``, with a simple majority necessary for approval.
+* A majority of a country's ``committers`` participation is necessary for a quorum on all voting matters.
```

Comment:
If a country gets a large number of committers, it may be impractical to achieve a majority – and we may end up with the same situation that this OCDEP is trying to solve, which is quicker PR feedback and merging. Could we have "A majority, or #, of a country's `committers` (whichever is smaller)", where `#` is some fixed number?